decidim-core/decidim-core.gemspec

```diff
@@ -50,7 +50,7 @@ Gem::Specification.new do |s|
   s.add_dependency "omniauth-rails_csrf_protection", "~> 1.0"
   s.add_dependency "omniauth-twitter", "~> 1.4"
   s.add_dependency "paper_trail", "~> 12.0"
-  s.add_dependency "pg", "~> 1.1.4", "< 2"
+  s.add_dependency "pg", "~> 1.3.0", "< 2"
```

Comment:
Just wondering, should we update this directly to `1.4.X` which has been released last week or is there some reason to keep it at `1.3.X`?